features/cellular/framework/AT/AT_CellularContext.cpp

```diff
@@ -374,6 +374,7 @@ bool AT_CellularContext::get_context()
                     continue;
                 }
 
+                //AnttiK: Redesign following 5 lines!
```

Comment:
This shouldn't be in production code

Reply:
Capturing notes like this as issues out in the open would be nice too.
Any context given to the community via issues will increase the likelihood of others being able to contribute outside of ARM.

Reply:
removed